rcljava/src/main/cpp/org_ros2_rcljava_RCLJava.cpp

```diff
@@ -58,20 +59,73 @@ Java_org_ros2_rcljava_RCLJava_nativeCreateNodeHandle(
 
   rcl_context_t * context = reinterpret_cast<rcl_context_t *>(context_handle);
 
+  // TODO(clalancette): we could probably make this faster by just doing these
```

Comment:
This can be done by using [JNI_onload](https://docs.oracle.com/javase/8/docs/technotes/guides/jni/spec/invocation.html#JNJI_OnLoad) (see also [GetEnv](https://docs.oracle.com/javase/8/docs/technotes/guides/jni/spec/invocation.html#GetEnv)).
The looked up methods need to be stored on global variables.

P.S.: It's fine to keep this as a TODO.

Reply:
Yeah, I'm going to leave it for now.  We do this elsewhere in the codebase, and I don't really expect users will call `createNode` that often.